### ISSUE TITLE
001 - Accessibility: screen reader doesn't read text in question selector on the About page

### ISSUE THREAD

**Issue by karlystark** [karlystark/portfolio#1]
When on the about page, question buttons are clickable and focusable, but the revealed text is not read aloud when each button is clicked. Look into Aria-expanded/hidden to see if screen reader registers state changes. Be sure to manage focus on text field when button is clicked too - see about adding a ref to the text area and using focus() method.
